app/views/layouts/groups.html.erb

```diff
@@ -22,6 +22,11 @@
             icon: "users",
             label: t(:"groups.sidebar.members")
           ) %>
+          <%= render section.with_item(
+            url: group_samples_path(@group),
```

Comment:
Can you move this above members as I think it is more important, I think you just copied what I did on the project page, and I will have to update that one.

Reply:
Please see aa83594e8438e4305598762175edf9914e0d2527.